context/context_test.go

```diff
@@ -105,7 +161,7 @@ func TestDependsOnNoCloserAllocation(t *testing.T) {
 	assert.Nil(t, ctx.finalizeables)
 }
 
-func TestDependsOn(t *testing.T) {
+func TestDependsOnX(t *testing.T) {
```

Comment:
Did you mean to change this method name?  Probably can remove the "X".

Reply:
Will revert back - was just testing stuff.